run/README.namelist

```diff
@@ -832,15 +797,15 @@ Namelist variables for controlling the adaptive time step option:
                                      = 3, Grell-Freitas ensemble scheme
                                      = 4, Scale-aware GFS Simplified Arakawa-Schubert (SAS) scheme 
                                      = 5, Grell 3D ensemble scheme
-                                     = 6, Modifed Tiedtke scheme (ARW only)
+                                     = 6, Modifed Tiedtke scheme
                                      = 7, Zhang-McFarlane scheme from CAM5 (CESM 1_0_1)
-                                     = 10, Modified Kain-Fritsch scheme with trigger function based on PDFs (ARW only)
+                                     = 10, Modified Kain-Fritsch scheme with trigger function based on PDFs
                                      = 11, Multi-scale Kain-Fritsch scheme
                                      = 14, KIM Simplified Arakawa-Schubert scheme (KSAS) across gray-zone resolutions
                                      = 16, A newer Tiedtke scheme
                                      = 94, 2015 GFS Simplified Arakawa-Schubert scheme (HWRF) 
                                      = 95, Previous GFS Simplified Arakawa-Schubert scheme (HWRF) 
```

Comment:
How about HWRF?

Reply:
@jamiebresch Will remove them too.

Reply:
@jamiebresch I'm modifying to read '(from HWRF, use with caution)'. Since the schemes were not removed from WRF, and may be able to be used by ARW.